addons/web/static/src/webclient/icons.scss

```diff
@@ -75,3 +74,16 @@ $oi-sizes: (
     mask: url('/web/static/img/barcode.svg') center/contain no-repeat;
     background-color: $o-brand-primary;
 }
+
+// This is rtl language specific fix. It will flip the icons 180 degree
+// ----------------------------------------------------------------------------
+.o_rtl .oi.oi-chevron-left,
+.o_rtl .oi.oi-chevron-right,
+.o_rtl .oi.oi-arrow-down-left,
```

Comment:
Just for readability, I'd suggest to nest selectors since everything is supposed to descend the `.o_rtl` class here.
 
 ```scss
 .o_rtl {
     .oi-chevron-right,
     .oi-arrow-down-left [...]
}